### ISSUE TITLE
Custom Validation Set for tpot.fit

### ISSUE THREAD

**Issue by skrish13** [EpistasisLab/tpot#767]
How do i give a seperate validation set so that tpot evaluates pipelines using that instead of random split of trainset itself?

**Comment by weixuanfu** [comment]
Please check the `cv` parameter in [TPOT API](https://epistasislab.github.io/tpot/api/). You can merge the validation set  with the trainset for fitting in `tpot_obj.fit(X, y)` and then specify train/test splits via an iterable (see [the example](https://stackoverflow.com/questions/27097330/how-to-customize-sklearn-cross-validation-iterator-by-indices) for `GridSearchCV`)

**Comment by sahil2897** [comment]
Can you please explain a little more how to custom specify training and validation data sets. I don't want to use k-fold cv?

**Comment by weixuanfu** [comment]
You could try other split models like `cv=TimeSeriesSplit(n_splits=5)` or use indexes of samples based on the example for GrifSearchCV above.
